lib/__tests__/validate.test.ts

```diff
@@ -0,0 +1,112 @@
+import fs, { Stats } from 'fs-extra';
+import { validateHubl } from '../../api/validateHubl';
+import { walk } from '../fs';
+import { lint } from '../cms/validate';
+import { LintResult, Validation } from '../../types/HublValidation';
+
+jest.mock('fs-extra');
+jest.mock('../../api/validateHubl');
+jest.mock('../fs');
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
TypeScript was being overly strict and requiring that I mock out `fs.stat` and `fs.readFile` in their entirety. Opted for this solution instead.